WebRoot/h/calendar

```diff
@@ -213,9 +213,9 @@
     var zprint = function(){
         try{ var idex = 0; var c =""; while (idex <= zrc ) { if(document.getElementById("C"+idex).checked) { cid = document.getElementById("C"+idex).value; c += cid + ","; } idex++ ; } }catch(ex){ }
         if (${view eq 'list'}) {
-            window.open("/h/printcalendar?view=${view}&l=${checkedCalendars}&date=${fmtStartDate}&endDate=${fmtEndDate}&action=print&imc=true");
+            window.open("/h/printcalendar?view=${fn:escapeXml(zm:replaceAll(view,'javascript:',''))}&l=${checkedCalendars}&date=${fmtStartDate}&endDate=${fmtEndDate}&action=print&imc=true");
```

Comment:
Is there any case where `view` contains `Javascript:` or `JAVASCRIPT:` and do we need to remove these string from view ?

Reply:
escapeXml will handle such cases.